extractor/src/main/java/org/schabi/newpipe/extractor/services/bandcamp/extractors/BandcampStreamExtractor.java

```diff
@@ -245,8 +246,17 @@ public StreamType getStreamType() {
     }
 
     @Override
-    public StreamInfoItemsCollector getRelatedItems() {
-        return null;
+    public PlaylistInfoItemsCollector getRelatedItems() {
+
+        PlaylistInfoItemsCollector collector = new PlaylistInfoItemsCollector(getServiceId());
+
+        Elements recommendedAlbums = document.getElementsByClass("recommended-album");
```

Comment:
`document` is null in `BandcampRadioStreamExtractor`. You need to override this method.

Reply:
Thanks. I should've run the bandcamp tests before opening the PR. Fixed through 36718767215d5a12becbf3176d5e010a461a338b.